src/mgr/MgrClient.cc

```diff
@@ -327,6 +329,9 @@ void MgrClient::send_report()
   }
 
   report->osd_health_metrics = std::move(osd_health_metrics);
+
+  cct->_conf->get_config_bl(&report->config_bl);
```

Comment:
It feels a bit much to send this on every report from every service: the value_bl part in md_config_t is already sorta-stateful and explicitly reset when changes are made, so get_config_bl could return a bool to indicate whether anything changed (i.e. whether value_bl was rewritten), and then this function would only include the config in the report in that situation.